docs/source/sdks.rst

```diff
@@ -6,10 +6,9 @@ SDK API Reference
 Python
 ------
 
-.. toctree::
+- `Transaction Processor <sdks/python_sdk/processor.html>`_
 
-   sdks/python_sdk/processor.rst
-   sdks/python_sdk/sawtooth_signing.rst
+- `Signing <sdks/python_sdk/sawtooth_signing.html>`_
```

Comment:
Is it expected at this point that these links don't work in the build artifact? https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1802/1/artifact/docs/build/html/sdks.html

Reply:
The build artifacts have a few glitches, so I'm not surprised.  (See my response to Peter's comment for more info.)